compiler/rustc_mir/src/const_eval/fn_queries.rs

```diff
@@ -50,7 +50,7 @@ pub fn is_min_const_fn(tcx: TyCtxt<'_>, def_id: DefId) -> bool {
             None => {
                 if let Some(stab) = tcx.lookup_stability(def_id) {
                     if stab.level.is_stable() {
-                        tcx.sess.span_err(
```

Comment:
can we just `span_bug!` here now, or is it still reachable even if the stability error was reported?

Reply:
This function is called in `check_unsafety`, which runs before `MissingStabilityAnnotations`.